server/index.js

```diff
@@ -10,68 +11,83 @@ const parser = require('body-parser');
 const app = express();
 const PORT = 3000;
 
-mongoose.connect('mongodb://localhost/menu', { useNewUrlParser: true }).then(() => console.log('db connected'));
+//mongoose.connect('mongodb://localhost/menu', { useNewUrlParser: true }).then(() => console.log('db connected'));
 
-const MenuItem = require('./models/MenuItem.js');
+const MenuItem = require('./models/MenuItemSQL.js');
 
-app.use('/:menu', express.static(path.join(__dirname, '../client/dist')));
-app.use(parser.json());
-app.use(parser.urlencoded({ extended: false }));
+//app.use('/:menu', express.static(path.join(__dirname, '../client/dist')));
+//app.use(parser.json());
+//app.use(parser.urlencoded({ extended: false }));
 
+app.use(express.static((`${__dirname}/../client/dist`)));
+app.use('/:menu', express.static((`${__dirname}/../client/dist`)));
+app.use(express.json());
 
-app.get('/api/menu', function (req, res) {
-     MenuItem.getAllMenus()
-     .then(data => res.status(202).send(data))
-     .catch(err => console.log(`API Error:${err}`));
- });
 
-app.post('/api/menu/', function (req, res) {
-    MenuItem.createMenu(req.body)
-     .then(data => res.status(202).send(data))
-     .catch(err => console.log(`API Error:${err}`));
-   });
 
 app.get('/api/menu/:menu', function (req, res) {
-  const menu = parseInt(req.params.menu);
-  if (menu > 100) {
-    res.status(404).send();
-  } else {
-    MenuItem.getMenu({ menu })
-    .then(data => res.status(202).send(data))
-    .catch(err => console.log(`API Error:${err}`));
-  }
+  let menu = parseInt(req.params.menu);
```

Comment:
Is this the menu *id*, specially? If so, it might be clearer if the variable were named `menuId` or even just `id`.